core/storage/user/gae_models.py

```diff
@@ -2400,3 +2403,50 @@ def get_user_id_migration_policy():
         a temporary model, so it doesn't need to be migrated.
         """
         return base_models.USER_ID_MIGRATION_POLICY.NOT_APPLICABLE
+
+
+class PseudonymizedUserModel(base_models.BaseModel):
+    """Model for storing anonymized user IDs."""
+
+    @staticmethod
+    def get_deletion_policy():
+        """PseudonymizedUserModel contains only anonymous ids."""
```

Comment:
anonymous --> pseudonymous (these are two different concepts). Ditto in multiple places below.